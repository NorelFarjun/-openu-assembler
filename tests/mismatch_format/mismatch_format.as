.data,2,3,4
.string ab
LABEL .string "abc"

