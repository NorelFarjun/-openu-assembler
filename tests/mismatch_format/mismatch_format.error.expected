0: Incorrect line
1: missing string
2: unknown instruction 'LABEL'
Bad input file - not outputting
